apps/remix-ide/src/app/files/fileManager.ts

```diff
@@ -733,12 +737,12 @@ class FileManager extends Plugin {
     }
   }
 
-  syncEditor (path) {
+  async syncEditor(path) {
     var currentFile = this._deps.config.get('currentFile')
     if (path !== currentFile) return
     var provider = this.fileProviderOf(currentFile)
     if (provider) {
-      provider.get(currentFile, (error, content) => {
+      await provider.get(currentFile, (error, content) => {
```

Comment:
I'm not sure how helpful an `await` is here since `provider.get` uses a callback.